irods_docker_files/docker_cmds_utilities.py

```diff
@@ -133,7 +137,8 @@ def run_database(database_type, database_container, alias_name, network_name):
     database_alias = 'database.example.org'
     if database_type == 'oracle':
         database_alias = 'oracle.example.org'
-        run_cmd = ['docker', 'run', '-d', '--rm',  '--name', database_container, '--shm-size=1g', '-e', 'ORACLE_PWD=testpassword', 'oracle/database:11.2.0.2-xe']
+        run_cmd = ['docker', 'run', '-d', '--rm',  '--name', database_container, '-h', database_alias, '--shm-size=1g', '-e', 'ORACLE_PWD=testpassword', 'oracle/database:11.2.0.2-xe']
```

Comment:
can use 'database_image' variable from above?

Reply:
i have added the name of the oracle image in the configuration.py file just like other database image names.

Reply:
i see it up top - please use it in line 139